apps/vr-tests/src/stories/ButtonNext.stories.tsx

```diff
@@ -13,6 +15,14 @@ storiesOf('Button Next', module)
         .snapshot('hover', { cropTo: '.testWrapper' })
         .mouseDown('button')
         .snapshot('pressed', { cropTo: '.testWrapper' })
+        .executeScript(
+          "document.getElementsByClassName('testWrapper')[0].classList.add('ms-Fabric--isFocusVisible')",
+        )
+        .executeScript("document.getElementsByTagName('button')[0].focus()")
+        .snapshot('focus', { cropTo: '.testWrapper' })
```

Comment:
i don't see new screener screenshots for the `focus` step?

Reply:
What do you mean? If you click on the screener results they are there:

![image](https://user-images.githubusercontent.com/7798177/86069422-a522ba80-ba2e-11ea-981c-c1fce95adc68.png)

Reply:
I'm blind :). approved!